x/pki/keeper/msg_server_add_x_509_cert.go

```diff
@@ -99,5 +99,8 @@ func (k msgServer) AddX509Cert(goCtx context.Context, msg *types.MsgAddX509Cert)
 	// add to subject -> subject key ID map
 	k.AddApprovedCertificateBySubject(ctx, certificate.Subject, certificate.SubjectKeyId)
 
+	// add to subject key ID -> certificates map
+	k.AddApprovedCertificateBySubjectKeyID(ctx, certificate)
```

Comment:
Please add a test to 'handler_test.go' to ensure that this method correctly modifies this state